TestCases/compliance-level-3/9999-test/9999-test.dmn

```diff
@@ -0,0 +1,15 @@
+<dmn:definitions
+xmlns:dmn="https://www.omg.org/spec/DMN/20191111/MODEL/"
+xmlns="https://kiegroup.org/dmn/_248D5897-53B4-44BD-A638-36315396C5E6" 
+xmlns:feel="https://www.omg.org/spec/DMN/20191111/FEEL/" 
+id="_1916E719-7C4D-477F-BA29-3F7FBED9A4A7" name="check" 
+typeLanguage="https://www.omg.org/spec/DMN/20191111/FEEL/" 
+namespace="https://kiegroup.org/dmn/_248D5897-53B4-44BD-A638-36315396C5E6">
+  <dmn:extensionElements/>
+  <dmn:inputData id="_8013DD6B-1333-45F6-A894-5120BB874C83" name="InputData-1">
+    <dmn:variable id="_65E3383E-546A-4528-8B45-424357CAAA67" name="InputData-1"/>
+  </dmn:inputData>
+  <dmn:inputData id="_id2" name="InputData-2">
+    <dmn:variable id="_id2var" name="InputData-2"/>
```

Comment:
Showing GHA for `TCK Validation` FAILURE:

![image](https://user-images.githubusercontent.com/1699252/125309212-dae0cb80-e331-11eb-88e6-d69fd7897573.png)